src/optimizer-page/scan-results/BrokenLinkTable.tsx

```diff
@@ -63,103 +63,104 @@ type TableData = {
-                <LinksCol
-                  block={{ url: block.url, displayName: block.displayName || 'Go to block' }}
-                  href={link}
-                />
-              ),
-              status: (
-                <CustomIcon
-                  icon={lockedIcon}
-                  message1={messages.lockedLabel}
-                  message2={messages.lockedInfoTooltip}
-                />
-              ),
-            }));
+      ) {
+        const blockLockedLinks = block.lockedLinks.map((link) => ({
+          Links: (
+            <LinksCol
+              block={{ url: block.url, displayName: block.displayName || 'Go to block' }}
+              href={link}
+            />
+          ),
+          status: (
+            <CustomIcon
+              icon={lockedIcon}
+              message1={messages.lockedLabel}
+              message2={messages.lockedInfoTooltip}
+            />
+          ),
+        }));
 
-            acc.push(...blockLockedLinks);
-          }
+        acc.push(...blockLockedLinks);
+      }
 
-          if (
-            filters.externalForbiddenLinks
+      if (
+        filters.externalForbiddenLinks
             || (!filters.brokenLinks && !filters.externalForbiddenLinks && !filters.lockedLinks)
-          ) {
-            const externalForbiddenLinks = block.externalForbiddenLinks.map((link) => ({
-              Links: (
-                <LinksCol
-                  block={{ url: block.url, displayName: block.displayName || 'Go to block' }}
-                  href={link}
-                />
-              ),
-              status: (
-                <CustomIcon
-                  icon={ManualIcon}
-                  message1={messages.manualLabel}
-                  message2={messages.manualInfoTooltip}
-                />
-              ),
-            }));
+      ) {
+        const externalForbiddenLinks = block.externalForbiddenLinks.map((link) => ({
+          Links: (
+            <LinksCol
+              block={{ url: block.url, displayName: block.displayName || 'Go to block' }}
+              href={link}
+            />
+          ),
+          status: (
+            <CustomIcon
+              icon={ManualIcon}
+              message1={messages.manualLabel}
+              message2={messages.manualInfoTooltip}
+            />
+          ),
+        }));
 
-            acc.push(...externalForbiddenLinks);
-          }
+        acc.push(...externalForbiddenLinks);
+      }
 
-          return acc;
-        },
-        [],
-      )}
-      columns={[
-        {
-          key: 'Links',
-          columnSortable: false,
-          width: 'col-9',
-          hideHeader: true,
-        },
-        {
-          key: 'status',
-          columnSortable: false,
-          width: 'col-3',
-          hideHeader: true,
-        },
-      ]}
-    />
-  </Card>
-);
+      return acc;
+    },
+    [],
+  );
+
+  return (
+    <Card className="unit-card rounded-sm pt-2 pb-3 pl-3 pr-4 mb-2.5">
```

Comment:
the padding button is added because before the `Table` component use to have a margin button of `1rem` that `DataTable` doesn't. In this way the we keep the card and the table spaced.